src/lib/entities/ingredient.py

```diff
@@ -7,3 +7,10 @@ def __init__(self):
         self.id = None  # integer
         self.name = None  # string
         self.description = None  # string
+
```

Comment:
Could you try to make it more generic? So that it works with for other entities. Try to get all props from a class a add a code to compare them